gcp_variant_transforms/libs/variant_merge/variant_merge_strategy.py

```diff
@@ -20,7 +20,7 @@
 class VariantMergeStrategy(object):
   """Interface for a variant merge strategy."""
 
-  def get_merged_variants(self, variants):
+  def get_merged_variants(self, variants, key):
```

Comment:
nice catch! Could you please add 'key' to the args section as well? It's a string representing the key used to merge ``variants``.

Reply:
Done.